src/standalone/Snow/Snow.jl

```diff
@@ -212,8 +203,10 @@ clipped values are what are actually applied as boundary fluxes, and are stored
 """
 auxiliary_vars(::SnowModel) = (
     :q_l,
+    :κ,
     :T,
     :T_sfc,
+    :z,
```

Comment:
what are kappa and z? it would be nice to explain them in the docstring

Reply:
good call!